### PR TITLE
Update CLI error codes to be consistent with `connect`

### PR DESCRIPTION
**Pull request by jefferai**

### CODE REVIEW

File: internal/cmd/commands/database/init.go [hunk 1]
```diff
@@ -253,20 +253,20 @@ func (c *InitCommand) Run(args []string) (retCode int) {
 	c.srv.DatabaseUrl, err = config.ParseAddress(urlToParse)
 	if err != nil && err != config.ErrNotAUrl {
 		c.UI.Error(fmt.Errorf("Error parsing database url: %w", err).Error())
-		return 1
+		return 2
```
Comment (louisruch) [inline internal/cmd/commands/database/init.go:256]: 3?

**Review by louisruch — COMMENTED** [review]
nit: given we are standardizing return codes here, it would be nice if they could be made consts instead of magic numbers within the code.  This will just help future coding and debugging to quickly know what each means.  

Feel free to add this as a backlog task as well

File: internal/cmd/commands/database/init.go [hunk 2]
```diff
@@ -181,17 +181,17 @@ func (c *InitCommand) Run(args []string) (retCode int) {
...
 	if err := c.srv.SetupKMSes(c.UI, c.Config); err != nil {
 		c.UI.Error(err.Error())
-		return 1
+		return 2
 	}
 
 	if c.srv.RootKms == nil {
 		c.UI.Error("Root KMS not found after parsing KMS blocks")
-		return 1
+		return 2
```
[the remark this answers is not in the document]
Comment (jefferai) [reply internal/cmd/commands/database/init.go:194]: Good catch

File: internal/cmd/commands/database/init.go (hunk 1 above)
Comment (jefferai) [reply internal/cmd/commands/database/init.go:256]: Here too

**Comment by jefferai** [comment]
@louisruch great idea on the consts; I will make that change now.

**Comment by jefferai** [comment]
@louisruch done, PTAL

**Review by louisruch — APPROVED** [review]
LGTM

**Review by louisruch — APPROVED** [review]

**Pull request merged**